apps/frontend/src/components/form/FinancialAssistanceForm.tsx

```diff
@@ -113,6 +113,21 @@ export const HospitalInfoSection: React.FC = () => {
         onChange={(value: string) => handleStateSelectChange(value)}
       />
 
+      {isOtherStatesSelected && (
```

Comment:
Did you change something here or is this from another MR? If it's not yours lets delete it from this MR

Reply:
Removed these changes